src/monitoring_service/handlers.py

```diff
@@ -381,16 +381,19 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
         non_closing_signer=event.non_closing_participant,
     )
     if monitor_request is None:
+        log.error("MonitorRequest was lost or deleted before triggering", monitor_request=monitor_request)
         return
 
     channel = context.db.get_channel(
         token_network_address=monitor_request.token_network_address,
         channel_id=monitor_request.channel_identifier,
     )
     if channel is None:
+        log.error("Channel info was lost or deleted before triggering", monitor_request=monitor_request, channel=channel)
```

Comment:
Same as above, just put that the channel info isn't there, and logging channel doesn't make sense